features/works/external_works.feature

```diff
@@ -0,0 +1,17 @@
+@works
+
+Feature: External works are not hosted on the Archive
+
+  Scenario: Can see an external works
```

Comment:
I think you could do without the "an" here, since it's external works, plural. Other than that and Hound's comment about an extra space, :+1: